NEWS.md

```diff
@@ -16,7 +16,7 @@ argument, e.g. `f <- . %>% abs %>% mean(na.rm = TRUE)`.
 Three new operators are introduced for some special cases
 
 * Compound assignment pipe: `%<>%`
-* Tee pipe: `%T>%`
+* Tee assignment pipe: `%T>%`
```

Comment:
This should just be tee

Reply:
Right. I had some confusion as to the use of the word "assignment" as you can see in the conversation of issue #178. Following that conversation, I was wondering if the names:
```
pipe
assignment pipe
exposition pipe
tee pipe
```
might not be better???

Reply:
Sure, sounds good.

Reply:
Ok. Will update them all to that then.

Reply:
Done